package.json

```diff
@@ -31,6 +31,7 @@
       "identifier-types": "1.2",
       "instance-statuses": "1.0",
       "instance-types": "2.0",
+      "inventory": "10.10",
```

Comment:
please check if it's released already. if not, maybe it make sense to specify previous, 10.9 or something what is released already

Reply:
looks like it's released